### ISSUE TITLE
Zeroth order mass action jumps

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaDiffEq/DiffEqJump.jl#31]
Do we handle this in MassActionJump? @Msturroc

**Comment by isaacsas** [comment]
We need to handle this nicer through the `MassActionJump constructor`, but this works for now:

```
using DiffEqBase, DiffEqJump, Plots
rate = [2.0]
rs = [Vector{Pair{Int,Int}}()]
 ns = [[1 => 1]]
jump = MassActionJump(rate, rs, ns)
prob = DiscreteProblem([100],(0.,100.))
jump_prob = JumpProblem(prob, Direct(), jump)
sol = solve(jump_prob, SSAStepper())
plot(sol)
```

**Comment by isaacsas** [comment]
Closed by [#33](https://github.com/JuliaDiffEq/DiffEqJump.jl/pull/33)
